samcli/lib/delete/utils.py

```diff
@@ -0,0 +1,17 @@
+"""
+Utilities for Delete
+"""
+
+from samcli.lib.utils.hash import file_checksum
+from samcli.lib.package.artifact_exporter import mktempfile
+
+
+def get_cf_template_name(template_str, extension):
```

Comment:
do you think we do not have a method to generate the template name .. I believe we should maintain this logic in one place.

Reply:
I agree, I will modify the code so that the logic to get the template file name when the template file is deployed and deleted remains the same.